packages/bbui/src/ActionButton/ActionButton.svelte

```diff
@@ -109,19 +110,22 @@
     background: var(--spectrum-global-color-gray-300);
     border-color: var(--spectrum-global-color-gray-500);
   }
-  .noPadding {
```

Comment:
This order is changed, otherwise `noPadding` and `quiet` was not working as expected